pages/Profile/index.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
Avoid using wildcard, or *, imports in your code. This can clutter the namespace/packages we're using and potentially lead to problems later down the line. Typically, just import what you need!
```suggestion
import React from 'react';
```